org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/inlayhint/LSPLineContentCodeMining.java

```diff
@@ -91,4 +113,81 @@ private static org.eclipse.jface.text.Position toPosition(Position position, IDo
 		return new org.eclipse.jface.text.Position(start, 1);
 	}
 
+	@Override
+	public final Consumer<MouseEvent> getAction() {
+		return me -> {
+			String title= getLabel();
+			if (title != null && !title.isEmpty()) {
+				findLabelPart(me).ifPresent(labelPart -> {
```

Comment:
would that be a bit more readable by reducing the number of nested ifs?
```suggestion
				findLabelPart(me).map(InlayHintLabelPart::getCommand).map(Command::getCommand).ifPresent(command -> {
```